src/main/scala/Schema.scala

```diff
@@ -0,0 +1,62 @@
+package CSVValidation
+
+import com.fasterxml.jackson.databind.{JsonNode, ObjectMapper}
+import com.fasterxml.jackson.databind.node.ObjectNode
+import traits.JavaIteratorExtensions.IteratorHasAsScalaArray
+
+import java.net.URL
+import java.nio.file.Paths
+object Schema {
+  val objectMapper = new ObjectMapper()
+
+  def loadMetadataAndValidate(
+      uri: String,
+      outputErrors: Boolean = true
+  ): (Option[TableGroup], Option[String]) = {
+    try {
+      val jsonNode = objectMapper.readTree(Paths.get(uri).toFile)
```

Comment:
So this will handle both `file:///` URLs as well as web urls like "https://"?

Reply:
I dont think it will handle both of them as of now. It is intended to work with the metadata file as of now.

Reply:
As discussed, it just needs to be the local file path specified as a parameter to the console app. There's no need to be able to input http(s) uris at all at this point.

Reply:
This is Rob.... We've had another look at the code and it transpires that csvlint.rb *does* infact allow you to put a URL on the commandline and it validates said thing. 
TODO for @josepajay: make sure we have an issue to add this functionality later (we don't need it right now).